### PR TITLE
happy bday blog

### PR DESCRIPTION
**Pull request by linsun**
Slow wifi - will try run make gen when i get home.

- [ ] Ambient
- [x ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by linsun** [comment]
cc @istio/steering-committee

**Comment by linsun** [comment]
@justinpettit had a comment - please suggest changes.  Would like to get this merged tonight - as I'm on vacation tmr.

File: content/en/blog/2024/happy-7th-birthday/index.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+"We've been using Istio in production for years now, it’s a key component of our infrastructure allowing us to
+securely connect micro-services, and provide ingress/egress traffic management and 1st class observability," said
+Frédéric Gaudet, Senior SRE at BlablaCar. "The community is great and each release brings a lot of exciting features."
+
+## Amazing diversity across end users and vendors
+
+Over the past year, our community has observed tremendous growth in terms of both the number of contributing
+companies and the number of contributors. Recall that Istio had 500 contributors when Istio turned three years
+old? Istio now has had over 1,700 contributors in the past year. With the Microsoft Open Service Mesh team joining
+the Istio community, we are now the preferred service mesh not only for Google Cloud, Red Hat OpenShift, and
```
Comment (justinpettit) [inline content/en/blog/2024/happy-7th-birthday/index.md:93]:
```suggestion
the Istio community, we are now the preferred service mesh not only for Red Hat OpenShift, and
```

File: content/en/blog/2024/happy-7th-birthday/index.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+Below are the top contributing companies for the past year, with Solo.io, Google, and DaoCloud being the top
+three. While most of these companies are Istio vendors, Salesforce and Ericsson are end users running Istio in production!
+
+{{< image width="80%"
+    link="./contribution.png"
+    alt="Top Istio contributing companies for the past year!"
+    >}}
+
+Here are some thoughts from community leaders:
```
Comment (kfaseela) [inline content/en/blog/2024/happy-7th-birthday/index.md:105]:
```suggestion
Here are some thoughts from our community leaders:
```

File: content/en/blog/2024/happy-7th-birthday/index.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+"What makes Istio unique is the community full of developers, users, and vendors from all across the globe working
+together to make Istio the best and most powerful open service mesh in the industry," said Neeraj Poddar, Istio
+Technical Oversight Committee member and VP of Engineering at Solo.io. "It’s the strength of the community that
+has made Istio so successful and now under CNCF I look forward to seeing Istio as the de facto service mesh
+standard for all cloud native applications."
+
+## Continuous Technical Innovation
+
+We are firm believers that diversity drives innovation. What amazes me most is the continuous innovation from the
```
Comment (kfaseela) [inline content/en/blog/2024/happy-7th-birthday/index.md:142]: What amazes "me"--> is that correct here?

**Comment by kfaseela** [comment]
Thanks @linsun ! You have captured a lot of info :)

File: content/en/blog/2024/happy-7th-birthday/index.md (hunk 3 above)
Comment (linsun) [inline content/en/blog/2024/happy-7th-birthday/index.md:142]:
```suggestion
We are firm believers that diversity drives innovation. What amazes us most is the continuous innovation from the
```

File: content/en/blog/2024/happy-7th-birthday/index.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+lifecycle, increase the stability of live environments by reducing flakiness, and build confidence in automated
+production deployments. Ultimately, this has accelerated the development process and improved the success rate of production deployments."
+
+"Istio enhances the security of our cloud platform while simplifying observability by integrating distributed
+tracing and OpenTelemetry," said Sathish Krishnan, Distinguished Engineer at UBS. "This combination provides
+robust security features and deep insights into system performance, enabling more effective monitoring and
+troubleshooting of our distributed services."
+
+"Adopting Istio has been a game changer for our engineering organization in our journey of adopting a
+microservices based architecture," said Sjray Kumar, Principal Software Engineer at Bluecore. "Its *batteries
```
Comment (linsun) [inline content/en/blog/2024/happy-7th-birthday/index.md:59]:
```suggestion
microservices based architecture," said Shray Kumar, Principal Software Engineer at Bluecore. "Its *batteries
```

File: .spelling [hunk 5]
```diff
@@ -1038,11 +1055,13 @@ Sidecarless
 SignalFX
 sigstore
 sinkInfo
+Sjray
```
Comment (linsun) [inline .spelling:1058]:
```suggestion
Shray
```

File: content/en/blog/2024/happy-7th-birthday/index.md (hunk 3 above)
Comment (linsun) [reply content/en/blog/2024/happy-7th-birthday/index.md]: corrected

File: content/en/blog/2024/happy-7th-birthday/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (linsun) [reply content/en/blog/2024/happy-7th-birthday/index.md]: sharp eye!!

File: content/en/blog/2024/happy-7th-birthday/index.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
+---
+title: "Happy 7th Birthday, Istio!"
+description: We are excited about Istio’s momentum and exciting future.
```
Comment (craigbox) [inline content/en/blog/2024/happy-7th-birthday/index.md:3]:
```suggestion
description: We celebrate Istio’s momentum and exciting future.
```

**Comment by chaltenio** [comment]
@linsun as per my request in the shared google doc (Istio momentum blog quotes), I propose please including a section with quotes from the founding team to acknowledge, celebrate and thanks their initial contributions. (Louis, Varun, Sven)

**Comment by chaltenio** [comment]
@craigbox @linsun @kfaseela are you missing @ZackButcher quote? it's in the document. Please add. 

Zack Butcher (Tetrate)
"It’s been a privilege to be part of the Istio project from the start, seeing it and the community mature and grow over the years. On a personal note, Istio has been central to my own career for the past eight years! I firmly believe that the best of Istio is yet to come, and in the coming years we’ll see continued growth, maturity, and adoption. Cheers to the wonderful community for reaching this milestone together."


Also please find below Varun's quote on behalf of the founding team -  no need for titles and company associations

"Istio was designed from the experience of OnePlatform at Google and we were confident of its fitment in Kubernetes from the start. It is great to see the momentum in the project in the last 7 years and we wish everyone in the community continues to work towards project growth and health and that it continues to have happy users and contributors."

**Review by craigbox — APPROVED** [review]

**Pull request merged**